src/_press/four-northern-california-transit-agencies-join-forces-to-buy-contactless-open-loop-fare-payment-systems-off-of-californias-purchasing-agreements.md

```diff
@@ -0,0 +1,36 @@
+tags:
+  - Contactless Payments
+outlet: "Cal-ITP"
+---
+
+Humboldt Transit Authority, Lake Transit Authority, Mendocino Transit Authority, and Redwood Coast Transit Authority—collectively part of the Far North Group—are the first group of agencies to band together to use the State of California’s competitively priced contracts to purchase and install the hardware and software services needed for accepting customers’ debit and credit cards and mobile wallets for fare payments.
+
+Cal-ITP—Caltrans’ California Integrated Travel Project—supported the agencies through the process of purchasing open-loop payment acceptance devices and fare calculation software from the State of California’s competitively awarded [Master Service Agreements](https://www.calitp.org/assets/Contactless.Payments.MSA.pdf?utm_source=Press+Release&utm_medium=Website&utm_id=FNG) (MSAs), contracts that allow U.S. transit agencies to purchase directly from vendors without further competitive bidding.
+
+“We’re excited to offer contactless fare payment to the riders in our region, to make it easy for customers to choose public transportation and pay for bus rides the same way they do for other everyday purchases,” says Greg Pratt, General Manager of Humboldt Transit Authority. “California’s MSAs and Cal-ITP’s free technical support helped us save the hassle and expense of running our own procurement while modernizing our fare systems with contactless hardware and fare-calculation and payment-processing software services. And by purchasing together from the MSAs, we were able to take advantage of bulk-buy cost savings.”
+
+No matter which of the four agencies’ fixed-route buses or dial-a-ride vehicles they board, customers will see the same contactless payment readers installed at the front of the bus. If a customer’s debit or credit card has the contactless symbol on it—or if they use a mobile payment option—they can tap the payment reader, see payment confirmation via a green checkmark on the screen, and then enjoy their ride. In addition to contactless-enabled bank cards, Apple Pay, Google Pay, Samsung Pay, and Fitbit Pay digital wallets can be tapped on the payment readers.
+
+“Paying for travel should be as simple as buying a cup of coffee, and adding contactless debit and credit card readers makes transportation more accessible and user-friendly for all riders. It also facilitates the seamless connections that travelers want and need,” says Gillian Gillett, Caltrans’ Data & Digital Services Division Chief and California Integrated Mobility Program Manager. Gillett leads [Cal-ITP](https://www.calitp.org/?utm_source=Press+Release&utm_medium=Website&utm_id=FNG), a California-based initiative to make riding by bus and rail simpler and more cost-effective—for public transportation providers and riders—by standardizing trip-planning information and modernizing fare collection systems.
```

Comment:
Notice the `?utm_source=Press+Release&utm_medium=Website&utm_id=FNG` part in the URL